shared/models/errors/advanced.yml

```diff
@@ -0,0 +1,62 @@
+type: string
+enum:
+  - address_length_exceeds_limit
+  - bank_account_already_verified
+  - bank_error
+  - custom_envelope_inventory_depleted
+  - deleted_bank_account
+  - failed_deliverability_strictness
+  - file_pages_below_min
+  - file_pages_exceed_max
+  - file_size_exceeds_limit
+  - foreign_return_address
+  - inconsistent_page_dimensions
+  - invalid_bank_account
+  - invalid_bank_account_verification
+  - invalid_check_international
+  - invalid_country_covid
+  - invalid_file
+  - invalid_file_dimensions
+  - invalid_file_download_time
+  - invalid_file_url
+  - invalid_image_dpi
+  - invalid_international_feature
+  - invalid_perforation_return_envelope
+  - invalid_template_html
+  - merge_variable_required
+  - merge_variable_whitespace
+  - payment_method_unverified
+  - pdf_encrypted
+  - special_characters_restricted
+  - unembedded_fonts
+description: |
+  A pre-defined string identifying Lob-specific errors
+  * `address_length_exceeds_limit` - 422: the sum of to.address_line1 and to.address_line2 cannot surpass 50 characters.
+  * `bank_account_already_verified` - 422: the bank account has already been verified.
```

Comment:
If I'd known you were working on this, I'd point you to the Errors section that Sid put in recently...

Reply:
Yeah, that was actually something I had intended to put in this PR way back when. This is separate, though, because this is the actual resource as opposed to just the documentation (I also wrote this months ago).